### ISSUE TITLE
Unsupported Extension OverloadedRecordFieds

### ISSUE THREAD

**Issue by huseyinyilmaz** [commercialhaskell/intero#133]
Hello,
I want to use OveloadedRecordFields in my project but intero gives following error message. I was wondering if you know a way to pass this error.

```
The following GHC options are incompatible with GHCi and have not been passed to it: -threaded
Configuring GHCi with the following packages: dummy
Intero 0.1.15 (GHC 7.10.3)
Type :intro and press enter for an introduction of the standard commands.

intero: unrecognised flag: -XOverloadedRecordFields

Usage: For basic information, try the `--help' option.

---

This is the buffer where Emacs talks to intero. It's normally hidden,
but a problem occcured.

TROUBLESHOOTING

It may be obvious if there is some text above this message
indicating a problem.

The process ended. Here is the reason that Emacs gives us:

  exited abnormally with code 1

For troubleshooting purposes, here are the arguments used to launch intero:

  stack ghci --with-ghc intero "--docker-run-args=--interactive=true --tty=false" --no-load --no-build --ghci-options -odir=/var/folders/7n/6km7q29d5bd38gtgzsj03jv80000gp/T/intero19980Ros --ghci-options -hidir=/var/folders/7n/6km7q29d5bd38gtgzsj03jv80000gp/T/intero19980Ros dummy

WHAT TO DO NEXT

If you fixed the problem, just kill this buffer, Intero will make
a fresh one and attempt to start the process automatically as
soon as you start editing code again.

If you are unable to fix the problem, just leave this buffer
around in Emacs and Intero will not attempt to start the process
anymore.

You can always run M-x intero-restart to make it try again.

```

**Comment by mgsloan** [comment]
OverloadedRecordFields requires GHC 8.  You should pick a resolver that uses ghc 8, like a recent nightly.

**Comment by huseyinyilmaz** [comment]
@mgsloan Thanks for the help. I solved my problem and wanted to give feedback in case somebody else end up here.

So I changed my resolver to resolver: nightly-2016-06-21
After that, I got following error.

```
following language extensions which are not supported by ghc-8.0.1:
OverloadedRecordFields
```

 So for some reason OverloadedRecordFields is not  supported by ghc-8.0.1. From that page:
```
https://ghc.haskell.org/trac/ghc/wiki/Records/OverloadedRecordFields
```
I learned that OverloadedRecordFields is implemented in 3 parts. and 2 of them will be in ghc 8
So, instead of OverloadedRecordFields extension, I used DuplicateRecordFields extension which was the one I needed.

And that work without a problem.

@mgsloan
Thanks again for the feedback.
You saved me hours of frustration.
